src/screens/QRCodeScannerScreen/BarCodeMask.styles.ts

```diff
@@ -2,69 +2,105 @@ import { StyleSheet, Dimensions } from 'react-native';
 import * as colors from '@util/colors';
 
 const { width, height } = Dimensions.get('screen');
-const windowSquare = 200;
+const windowSquare = 300;
 
+/*
+	Scaler for Offset in Y-axis based on windowSquare.
+	Used in multiple cases.
+	Can be cleaned up.
+*/
+const scaler = windowSquare / (windowSquare / 50);
 const widthMinusWindow = (width - windowSquare) / 2;
-const heightMinusWindow = (height - windowSquare) / 2;
+const heightMinusWindow = ((height - windowSquare) / 2) - scaler;
 
 export default StyleSheet.create({
-	top: {
-		position: 'absolute',
-		backgroundColor: colors.DARK_GRAY_TRANSPARENT,
-		top: 0,
-		left: 0,
-		width,
-		height: heightMinusWindow,
-	},
-	bottom: {
-		position: 'absolute',
+	background: {
 		backgroundColor: colors.DARK_GRAY_TRANSPARENT,
-		bottom: 0,
-		left: 0,
+		height,
 		width,
-		height: heightMinusWindow,
-	},
-	left: {
-		position: 'absolute',
-		backgroundColor: colors.DARK_GRAY_TRANSPARENT,
-		top: heightMinusWindow,
-		left: 0,
-		width: widthMinusWindow,
-		height: windowSquare,
-	},
-	right: {
-		position: 'absolute',
-		backgroundColor: colors.DARK_GRAY_TRANSPARENT,
-		top: heightMinusWindow,
-		right: 0,
-		width: widthMinusWindow,
-		height: windowSquare,
+		zIndex: -1,
 	},
 	xContainer: {
 		position: 'absolute',
-		top: 30,
-		left: 10,
-		justifyContent: 'center',
-		alignItems: 'center',
+		display: 'flex',
+		flexDirection: 'row',
+		top: 0,
+		left: 0,
+		justifyContent: 'space-between',
+		width: '100%',
+		paddingTop: 20,
+		paddingHorizontal: 5,
 	},
 	x: {
 		textAlign: 'center',
-		textAlignVertical: 'center',
-		fontSize: 70,
+		fontSize: 48,
 	},
 	textContainer: {
 		position: 'absolute',
```

Comment:
I guess it's not good to use absolute position there, it does not compatible with flexbox, and I think that's the reason of misalignment in my IOS simulator....

Reply:
OOPS -- Thanks Tao! I'll look into it :)